coralproject/talk
# Load more replies

## What does this PR do?

Allows the user to load additional replies if more than 3 are listed on a comment. 

## How do I test this PR?

1. Create a comment with fewer than 3 replies. 
2. "View More" button should not appear.
3. Create more than 3 replies.
4. "View More" button should appear.
5. Click "View More" button.
6. Up to 10 additional replies should appear.
7. If there are more than 10, the "View More" button should keep appearing until all replies are displayed.

## Note

All E2E tests pass locally.
